.github/workflows/periodic-ci.yml

```diff
@@ -81,13 +81,10 @@ jobs:
         with:
           go-version-file: .go-version
 
-      - name: Fetch OPA version
-        run: echo "OPA_VERSION=$(go list -m -f {{.Version}} github.com/open-policy-agent/opa | sed 's/v//')" >> $GITHUB_ENV
-
       - name: Setup OPA
         uses: open-policy-agent/setup-opa@v2
         with:
-          version: ${{ env.OPA_VERSION }}
+          version: 0.44 # this should be aligned with the version in go.mod
```

Comment:
In `go.mod` we are using `0.44.1-data-data`, when stripping the data we still have `0.44.1` which does not exists.
We can remove that as well but it will be error prune once again.